kda/tests/test_kda.py

```diff
@@ -848,9 +848,21 @@ def test_sigma_K_4WL(self, k_vals):
     @pytest.mark.parametrize(
         "cycle, cycle_order, expected_func",
         [
-            ([0, 3, 2, 1], [3, 0], "log(k12*k23*k34*k41/(k14*k21*k32*k43))",),
-            ([0, 3, 1], [3, 0], "log(k12*k24*k41/(k14*k21*k42))",),
-            ([1, 3, 2], [3, 1], "log(k23*k34*k42/(k24*k32*k43))",),
+            (
+                [0, 3, 2, 1],
+                [3, 0],
+                "log(k12*k23*k34*k41/(k14*k21*k32*k43))",
+            ),
+            (
+                [0, 3, 1],
+                [3, 0],
+                "log(k12*k24*k41/(k14*k21*k42))",
+            ),
+            (
+                [1, 3, 2],
+                [3, 1],
+                "log(k23*k34*k42/(k24*k32*k43))",
+            ),
```

Comment:
These changes seem to be stemming from an inconsistency between running `black` in different environments. They probably should be removed from this PR but I might let them stay just for the sake of time.

Reply:
Okay, so this is an issue with the `python-black` package in `Atom`.  `black kda/tests/test_kda.py` brings this back to the original configuration, so I'll reformat before pushing the next version.